frontend/src/algolia-widgets.js

```diff
@@ -0,0 +1,49 @@
+/*!
+ * sri sri guru gaurangau jayatah
+ */
+
+import { connectInfiniteHits } from 'instantsearch.js/es/connectors';
+import Hogan from 'hogan.js';
+import { dom } from './dom-utils';
+import { TempStorage } from './temp-storage';
+
+export class AlgoliaWidgets {
+  get appInfiniteHits() {
+    const renderInfiniteHits = (renderOptions) => {
```

Comment:
I don't think overriding rendering of the `infiniteHits` widget is necessary and justified for this task.

Reply:
as i have mentioned in previous comments that there is a use case where we need to restore default UI and hide error message. so this particular widget is written to achieve same.